### ISSUE TITLE
Record on movement through UI not remembered

### ISSUE THREAD

**Issue by megapearl** [seydx/homebridge-camera-ui#432]
**Describe the bug**
When switching on 'record on movement through UI' and save using save button lower right, and restarting camera.ui (using homebridge OR settings -> system -> restart in camera.ui interface) the setting is not remembered and is OFF again.

**To Reproduce**
Swith on 'record on movement through UI' save the setting and restart camera.ui.

**Expected behavior**
The setting should be still on after restarting camera.ui

**Logs**

**Screenshots**
<img width="994" alt="image" src="https://user-images.githubusercontent.com/3420660/152174376-50e83151-3f88-4afe-a692-924860c52b85.png">

**Environment**
- Node.js Version: v16.13.2
- Homebridge Version: v1.4.0 (HAP v0.10.0)
- Camera UI Version: v5.0.16-latest
- Operating System: Debian 10 with Docker Alpine Linux (3.12.9)
- Browser Version: Edge 97.0.1072.76 (Official build) (64-bit) on Windows 11

**Comment by Ulisestts** [comment]
I am also having the same issue, already reinstalled everything and still occurs. Any solutions?

**Comment by ThatsNiceGuy** [comment]
I am also having this issue, with just one camera attached and very simple settings.

Node v16.14.2
NPM v8.5.0
homebridge-camera-ui v5.0.16
Homebridge v1.4.0

Output of `uname -a`:
```
Linux hostname 5.10.0-0.bpo.12-amd64 #1 SMP Debian 5.10.103-1~bpo10+1 (2022-03-08) x86_64
```

Homebridge installed via [these](https://github.com/homebridge/homebridge/wiki/Install-Homebridge-on-Debian-or-Ubuntu-Linux) instructions. No Docker involved.

<details>
  <summary>Homebridge configuration</summary>
  
  ```json
  {
    "bridge": {
        "name": "ThatsNiceGuy Homebridge XXXX",
        "username": "XX:XX:XX:XX:XX:XX",
        "port": 51557,
        "pin": "XXX-XX-XXX"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "name": "CameraUI",
            "port": 8582,
            "debug": false,
            "mqtt": {
                "active": false,
                "tls": false,
                "port": 1883
            },
            "http": {
                "active": false,
                "port": 7272,
                "localhttp": false
            },
            "smtp": {
                "active": false,
                "port": 2727,
                "space_replace": "+"
            },
            "ftp": {
                "active": false,
                "useFile": false,
                "port": 5050
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "Front Porch",
                    "motionTimeout": null,
                    "prebuffering": true,
                    "videoConfig": {
                        "source": "-i rtsp://192.168.1.199:554/h264",
                        "stillImageSource": "-i rtsp://192.168.1.199:554/h264cif",
                        "stimeout": null,
                        "audio": null,
                        "debug": null,
                        "subSource": "-i rtsp://192.168.1.199:554/h264cif",
                        "maxStreams": null,
                        "maxWidth": 1920,
                        "maxHeight": 1080,
                        "maxFPS": 30,
                        "maxBitrate": 5120,
                        "rtspTransport": null,
                        "vcodec": "libx264",
                        "acodec": null
                    },
                    "mqtt": {
                        "motionTopic": null,
                        "motionMessage": null,
                        "motionResetTopic": null,
                        "motionResetMessage": null
                    },
                    "smtp": {
                        "email": "Front Porch"
                    },
                    "videoanalysis": {
                        "active": true
                    },
                    "unbridge": true,
                    "motion": null,
                    "doorbell": null,
                    "switches": null,
                    "motionDoorbell": true,
                    "excludeSwitch": null,
                    "privacySwitch": null,
                    "manufacturer": "ThatsNiceGuy",
                    "model": "ANPVIZ IPC-D230W",
                    "serialNumber": "XXXXXXXXXXX",
                    "prebufferLength": 4,
                    "hsv": false,
                    "subtype": "camera"
                }
            ],
            "platform": "CameraUI",
            "atHomeSwitch": false,
            "options": {
                "videoProcessor": "/usr/lib/node_modules/homebridge-camera-ui/node_modules/ffmpeg-for-homebridge/ffmpeg"
            }
        }
    ]
  ```

</details>

<details>
  <summary>camera.ui configuration</summary>
  
  ```json
  {
  "debug": false,
  "port": 8582,
  "atHomeSwitch": false,
  "options": {
    "videoProcessor": "/usr/lib/node_modules/homebridge-camera-ui/node_modules/ffmpeg-for-homebridge/ffmpeg"
  },
  "ssl": {
    "active": false
  },
  "http": {
    "active": false,
    "port": 7272,
    "localhttp": false
  },
  "smtp": {
    "active": false,
    "port": 2727,
    "space_replace": "+"
  },
  "ftp": {
    "active": false,
    "useFile": false,
    "port": 5050
  },
  "mqtt": {
    "active": false,
    "tls": false,
    "port": 1883
  },
  "cameras": [
    {
      "name": "Front Porch",
      "motionTimeout": null,
      "prebuffering": true,
      "videoConfig": {
        "source": "-i rtsp://192.168.1.199:554/h264",
        "stillImageSource": "-i rtsp://192.168.1.199:554/h264cif",
        "stimeout": null,
        "audio": null,
        "debug": null,
        "subSource": "-i rtsp://192.168.1.199:554/h264cif",
        "maxStreams": null,
        "maxWidth": 1920,
        "maxHeight": 1080,
        "maxFPS": 30,
        "maxBitrate": 5120,
        "rtspTransport": null,
        "vcodec": "libx264",
        "acodec": null
      },
      "mqtt": {
        "motionTopic": null,
        "motionMessage": null,
        "motionResetTopic": null,
        "motionResetMessage": null
      },
      "smtp": {
        "email": "Front Porch"
      },
      "videoanalysis": {
        "active": true
      },
      "unbridge": true,
      "motion": null,
      "doorbell": null,
      "switches": null,
      "motionDoorbell": true,
      "excludeSwitch": null,
      "privacySwitch": null,
      "manufacturer": "ThatsNiceGuy",
      "model": "ANPVIZ IPC-D230W",
      "serialNumber": "XXXXXXXXXXX",
      "prebufferLength": 4,
      "hsv": false,
      "subtype": "camera"
    }
  ]
}
  ```

</details>

**Comment by seydx** [comment]
fixed in v5.0.17-beta.3
